### DISCUSSION TITLE
Compile configuration for help

### DISCUSSION THREAD

**Discussion by hcxmj** [webpack/webpack#16984] [Q&A]
Environment configuration version：
    "webpack": "^5.74.0",
    "webpack-cli": "^4.10.0",
    "webpack-dev-server": "^4.10.0",
    "webpack-merge": "^5.8.0",
    "webpackbar": "^5.0.2",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-image-webp": "^0.8.0",
    "react-intersection-observer": "^9.4.2",
    "react-router-dom": "^6.3.0",
    "@babel/core": "^7.18.10",
    "@babel/plugin-transform-modules-commonjs": "^7.18.6",
    "@babel/polyfill": "^7.12.1",
    "@babel/preset-env": "^7.18.10",
    "@babel/preset-react": "^7.18.6",
    "@babel/preset-typescript": "^7.18.6",
   "typescript": "*",

Problem Description：
When I use react+ts/tsx to develop a project, there is always es6 syntax in the packaged and compiled code, such as let/const or new Map/Promise. Part of it seems that the node dependency package itself contains es6 syntax, resulting in The white screen cannot be opened normally on the webview of the lower version of Android

Below is my configuration：
tsconfig.json:
`{
    "compilerOptions": {
      "lib": ["dom", "dom.iterable", "esnext"],
      "allowJs": true,
      "allowSyntheticDefaultImports": true,
      "skipLibCheck": true,
      "esModuleInterop": true,
      "strict": true,
      "forceConsistentCasingInFileNames": true,
      "moduleResolution": "node",
      "resolveJsonModule": true,
      "isolatedModules": true,
      "noEmit": true,
      "jsx": "react",
      "baseUrl": ".",
      "paths": {
        "@/*": ["./src/*"]
      },
    },
    "include": ["src", "./src/types/image.d.ts",],
    "exclude": ["node_modules"],
  }
  `
.babelrc:
{
    "presets": [
        "@babel/preset-react", 
        "@babel/preset-typescript",
        [
            "@babel/preset-env",
            {
              "targets": {
                "chrome": "49",
                "ios": "10"
              }
            }
        ]
    ]
}

webpack:
      {
        test: /\.(ts|js)x?$/i,
        exclude: /node_modules/,
        use: {
          loader: 'babel-loader',
          options: {
            presets: ['@babel/preset-env', '@babel/preset-react', '@babel/preset-typescript']
          }
        }
      },

I have been troubled by this problem for a long time and I can't solve it. I hope to get your help, thank you

**Comment by alexander-akait** [comment]
Remove `exclude: /node_modules/`, because your dependecies use es6
